apps/avatax/scripts/migrations/run-migration.ts

```diff
@@ -27,7 +26,7 @@ const runMigration = async () => {
 
       const webhookMigrator = createAppWebhookMigrator(env, { mode: "migrate" });
 
-      await migrateTaxes(webhookMigrator);
+      // TODO Run migration here
```

Comment:
I think we should throw an error or at least add some logs, in case someone tries to run the migrations